src/Cookie/SetCookie.php

```diff
@@ -207,6 +208,8 @@ public function setMaxAge($maxAge): void
 
     /**
      * The UNIX timestamp when the cookie Expires
+     *
+     * @return null|string|int
```

Comment:
According to [this](https://github.com/guzzle/guzzle/pull/2677/files#diff-6456b4cf27d0bc8bc517e1e5d36e0618R226)

```
 $this->data['Expires'] = \is_numeric($timestamp)
            ? (int) $timestamp
            : \strtotime($timestamp);
```

then most probably the `expires` key will always be an int right?

Reply:
Theoretically, no. Practically (after construction), yet.

* `__construct` does use both the getter and setter 
* `$defaults` defines `Expires` as `null`.

I eventually removed `int` as return typehint. 

Not calling the getter / setter in the constructor would allow removing `null`. Let me know if you want me to do that, I'll then create another PR for that.